### PR TITLE
Converted possible value of lr_flag to enum

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: both

* **Description of Pull Request**: 

Converted possible value of lr_flag (1: left hand weapon; 2: arrow; 3: shield) to enum.

### CODE REVIEW

**Review by Lemongrass3110 — APPROVED** [review]
Nice refactor!

Please change sd->state.lr_flag's type to e_lr_flag.

File: src/map/pc.hpp [hunk 1]
```diff
@@ -1064,7 +1071,7 @@ enum e_penalty_type : uint16{
 	PENALTY_MVP_DROP,
 	PENALTY_MAX
 };
-
+
```
Comment (Atemo) [inline src/map/pc.hpp:1074]:
```suggestion

```

File: src/map/pc.hpp [hunk 2]
```diff
@@ -122,6 +122,13 @@ enum e_additem_result : uint8 {
 	ADDITEM_STACKLIMIT
 };
 
+enum e_lr_flag : uint8 {
+	LR_FLAG_NONE = 0,
+	LR_FLAG_WEAPON,
+	LR_FLAG_ARROW,
+	LR_FLAG_SHIELD,
```
Comment (Atemo) [inline src/map/pc.hpp:129]:
```suggestion
	LR_FLAG_SHIELD
```

File: src/map/pc.hpp [hunk 3]
```diff
@@ -1064,7 +1071,6 @@ enum e_penalty_type : uint16{
 	PENALTY_MVP_DROP,
 	PENALTY_MAX
 };
-
 struct s_penalty{
```
Comment (Atemo) [inline src/map/pc.hpp:1074]:
```suggestion

struct s_penalty{
```

**Pull request merged**